### ISSUE TITLE
Review sediment loading variables output

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/gwlf-e#48]
Following https://github.com/WikiWatershed/gwlf-e/issues/39, review the output from the ported model and check for accuracy. Consult with BME to check accuracy of calculated variables. Determine approach for resolving discrepancies, if necessary.

**Comment by kdeloach** [comment]
I have observed a considerable amount of precision loss in the VB code. 

Since many variables are defined without a type, VB incorrectly infers the data type when loading these values from the summary dat files. For example, numbers that are read from a file, with less than 5 significant digits to the right of the decimal point, are automatically cast to the [Currency](https://msdn.microsoft.com/en-us/library/office/gg264338.aspx) data type. Variables that are initialized as `Currency` can only store 4 significant digits of precision. This means that the numbers in the Python version will be more precise, which will make comparing the Python output to the VB output much more difficult.

I've attached two files to demonstrate the differences in precision between the Python and VB version. These output files were generated by using the same GMS file in both versions of the model.

[gwlfe-output-python.txt](https://github.com/WikiWatershed/gwlf-e/files/260058/gwlfe-output-python.txt)
[gwlfe-output-vb.txt](https://github.com/WikiWatershed/gwlf-e/files/260059/gwlfe-output-vb.txt)
